changelog.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## sap.s4hana.api_inspectionlot_srv
+## sap.s4hana.api_inspectionplan_srv
```

Comment:
This should be added to end of file as spearate heading with its own sub headings